src/main/java/io/fabric8/jenkins/openshiftsync/PodTemplateUtils.java

```diff
@@ -143,7 +154,7 @@ public static synchronized void addPodTemplate(PodTemplate podTemplate) {
     // clear out existing template with same name; k8s plugin maintains
     // list, not map
     removePodTemplate(podTemplate);
-
+    podTemplate = setProxyEnvVar(podTemplate);
```

Comment:
return value is not needed here

Reply:
Here we arent using kubecloud api inside setProxyEnvVar to updat the variable. Makes more sense to have it I feel.

Reply:
Understool :)